src/main/java/org/broadinstitute/consent/http/resources/DaaResource.java

```diff
@@ -195,4 +199,23 @@ public Response deleteDaaForUser(
       return createExceptionResponse(e);
     }
   }
+
+  @POST
+  @PermitAll
+  @Path("/request/{daaId}")
+  public Response sendDaaRequestMessage(
+      @Auth AuthUser authUser,
+      @PathParam("daaId") Integer daaId) {
+    try {
+      User user = userService.findUserByEmail(authUser.getEmail());
```

Comment:
Here is another place where we should check the user's institution and return an error. I'm debating on what that is, but 403 is technically the right answer here ... we understand the request, but are refusing to honor it for internal reasons. However, I think a 400 is more consistent with how we are handling this in other scenarios.